mlflow/server/handlers.py

```diff
@@ -782,7 +782,7 @@ def _set_tag():
         schema={
             "run_id": [_assert_required, _assert_string],
             "key": [_assert_required, _assert_string],
-            "value": [_assert_required, _assert_string],
+            "value": [_assert_string],
```

Comment:
Shall we also update `_set_experiment_tag` ?

Reply:
Q:

> When parsing JSON payloads via proto, absent string fields are expressed as empty strings

So that we cannot distinguish whether the `value` is empty string or absent `value` fields, is this risky ?

Reply:
Good catch! I've updated `_set_experiment_tag`, `_set_registered_model_tag`, and `_set_model_version_tag`. I've also simplified tests.

I don't think that distinguishing between empty strings and absent values is necessary, since MLflow supports tags that have empty string values, and this behavior is easily explainable to the user. This has been the behavior for a long time (likely since the launch of MLflow).